ui/builder.js

```diff
@@ -133,7 +133,7 @@ const esbuildParams = {
     logTime(),
   ],
   define: genDefine(),
-  inject: ['./process-shim.js'], // fix runtime crash
+  // inject: ['./process-shim.js'], // fix runtime crash
```

Comment:
Test again after merging the master code, it seems it doesn't report the `process.cwd()` error when running if we comment this line code.